build.gradle

```diff
@@ -183,14 +185,15 @@ apply plugin: 'org.jetbrains.intellij'
 apply plugin: ChangeLogPlugin
 
 intellij {
-    version "IC-" + ideaVersion
+    version ideVersion("IC")
```

Comment:
Is it better to move the product code to properties file as well?

Reply:
Product code will be tied to the `jetbrains-*` module. I.E, `jetbrains-ult` needs to use IU